src/content/translations/pt-br/dapps/index.md

```diff
@@ -0,0 +1,33 @@
+---
+title: Uso de aplicações da Ethereum (dapps)
+description: As informações básicas necessárias para começar a usar a Ethereum.
+lang: pt-br
+---
+
+# Como usar a Ethereum {#using-ethereum}
+
+<div class="featured">
+
+**Quer começar a usar a Ethereum?** Existem centenas de aplicações descentralizadas (dapps) criadas na Ethereum que você pode começar a usar hoje.
+
+</div>
+
+## Como usar uma aplicação criada na Ethereum {#how-to-use-an-application-built-on-ethereum}
+
+A melhor maneira de começar com a Ethereum é começar a usá-la!
+
+Aqui estão apenas alguns apps construídos na Ethereum - nós alteramos esta lista com frequência!
+
+<RandomAppList />
+
+Interessado em ver mais aplicativos construídos na Ethereum?
+
+- [Built on Ethereum](https://docs.ethhub.io/built-on-ethereum/built-on-ethereum/) _Atualizado frequentemente - EthHub_
+- [90+ Ethereum Apps You Can Use Right Now](https://media.consensys.net/40-ethereum-apps-you-can-use-right-now-d643333769f7) _Abril 24, 2019 - ConsenSys_
+- [Ethereum Dapps](https://www.stateofthedapps.com/rankings/platform/ethereum) _Atualizado frequentemente - State of the Dapps_
+- [Ethereum DeFi Ecosystem](https://defiprime.com/ethereum) _Atualizado frequentemente - Defiprime_
+- [DeFi Pulse](https://defipulse.com/) _Análises + rankings de protocolos DeFi - Defi Pulse_
+
+Algumas aplicações Ethereum necessitam de uma carteira - [saiba mais sobre carteiras Ethereum aqui](/pt-br/pt-BR/wallets/).
```

Comment:
Looks like all these internal links are broken - they duplicate the lang path (e.g. `/pt-br/pt-BR/wallets/`).

Our internal linking is no intelligent enough to automatically add the language path, so you can replace all `/pt-br/pt-BR/` with just `/` (e.g. so a link to `/pt-br/pt-BR/wallets/` becomes `/wallets/`)